packages/shared/src/components/MainLayout.tsx

```diff
@@ -90,6 +99,26 @@ function MainLayoutComponent({
   useReferralReminder();
   useStreakMilestone();
 
+  const marketingCta = getMarketingCta(MarketingCtaVariant.Popover);
+
+  useEffect(() => {
```

Comment:
Because it is inside `onAfterOpen`, which is fired when the modal is opened.
Could do it the way it's done in the card, I was thinking about it, but I didn't want to throw in a ref 😬